### PR TITLE
MOYO Header Solution

### PR DESCRIPTION
**Pull request by WaveOfDandelions**
[HEADER](https://WaveOfDandelions.github.io/layout_moyo-header/)

[TEST](https://WaveOfDandelions.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,50 @@
...
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <img src="./images/logo.png" alt="logotype">
```
Comment (choeqq) [inline src/index.html:17]: I think we need `< a/>` tag around `img` - our logo probably links to main page 😉

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-is-hover: #00acdc;
+}
+
 body {
```
Comment (choeqq) [inline src/style.css:6]: I would probably provide more generous CSS styles reset like:
```
*, *::after, *::before {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```
but that's a good start 😄 

[more about custom CSS resets](https://www.joshwcomeau.com/css/custom-css-reset/)

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
```
Comment (choeqq) [inline src/style.css:38]: we can also just remove margin from last child 😄

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,69 @@
+:root {
```
Comment (choeqq) [inline src/style.css:1]: props for using CSS variables 👍

**Review by choeqq — APPROVED** [review]
It would be useful to provide the TEST and HEADER links in PR description, I updated it for you 😉

File: src/style.css (hunk 2 above)
Comment (WaveOfDandelions) [reply src/style.css:6]: Is it better to use normalize css for that?

Comment (choeqq) [reply src/style.css:6]: For this kind of project I would stick to the basics, and those simple styles are more than enough, but for larger projects I would definitely take that into consideration 😉

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**